R/assess_export_help.R

```diff
@@ -36,3 +36,36 @@ assess_export_help.pkg_install <- function(x, ...) {
 score.pkg_metric_export_help <- function(x, ...) {
   sum(x, na.rm = TRUE) / length(x)
 }
+
+
+
+#' Provide development hints for improving exported value documentation
```

Comment:
For this case you're right, @elong0527, but for many other metrics that might not be the case.

This was just a quick experiment about the possibility of extending the package with these types of tips. This one isn't particularly useful because it's redundant with R CMD CHECK, but we'll definitely collect metrics that wouldn't be captured that way where it might be suitable. For example, the NEWS document being up to date or the community engagement and package maintenance won't be found via R CMD check, and it would be nice to be transparent to package authors about what they can do to improve their score. I see this as a nice-to-have, but not a mandatory part of riskmetric.